src/jsonld/article.tsx

```diff
@@ -6,7 +6,7 @@ import { setPublisher } from 'src/utils/schema/setPublisher';
 import { ArticleAuthor } from 'src/types';
 
 export interface ArticleJsonLdProps extends JsonLdProps {
-  type?: 'Article' | 'Blog' | 'NewsArticle';
+  type?: 'Article' | 'BlogPosting' | 'NewsArticle';
```

Comment:
```suggestion
  type?: 'Article' | 'BlogPosting' | 'NewsArticle' | 'Blog';
```

🤔 This is a tricky one. If we make this change it will cause typescript builds to fail. So I would likley have to mark it as a breaking change.

Can we leave `Blog` there for now and add `BlogListing` 

All the docs can just refer to `BlogPosting`

I think this is the only change required.

Reply:
Sure! 😉